packages/react/src/components/Popover/Popover.mdx

```diff
@@ -19,13 +19,53 @@ import { Preview, Props, Story } from '@storybook/addon-docs/blocks';
 
 **Note: this component is currently experimental and does not follow semver.**
 
-The Popover component is used to build layers that appear on top of the current
-surface.
+A popover is a UI element that _pops over_ or renders on top of other UI on the
+page. It's a component "primitive" meaning it can be used to help make more
+complex components that utilize this behavior. It is often styled as what we'd
+traditionally consider a tooltip.
+
+## Usage 
+
+```jsx
+import { popover } from 'carbon-components-react';
+
+function MyPopover() {
+  return (
+    <TriggeringUiElement/>
+    <Popover 
+      open={open} 
+      align={top} 
+      relative>
+      <PopoverContent className="custom-classname"/>
+    </Popover>
+  )
+}
+```
 
 ## Component API
 
 <Props />
 
+## Caret Alignment 
+
+Users can describe the caret alignment in one of twelve directions. Caret alignement dictates position and direction of the popover.
+
+- 'top',
```

Comment:
Instead of a list, could we instead give a couple of examples of configuration?

Something like:

> For example, when you use `align="top"` the Popover component will align to the top of the trigger element. Likewise, `align="bottom"` will align the Popover component to the bottom of the trigger element.
>
> You can also configure the caret position by using `top-left` or `top-right`, and `bottom-left` or `bottom-right`. This will place the caret on either the left or right side of the Popover. By default, the caret will render in the middle of the Popover.
>
> These rules also apply to `left` and `right` alignment, but instead you will. use `left-top` and `left-bottom`, or `right-top` and `right-bottom`, to specify the alignment of the caret in the Popover.